libs/server-sdk/tests/server_c_bindings_test.cpp

```diff
@@ -52,9 +52,11 @@ TEST(ClientBindings, RegisterDataSourceStatusChangeListener) {
     LDListenerConnection connection =
         LDServerSDK_DataSourceStatus_OnStatusChange(sdk, listener);
 
```

Comment:
This change is an implication of the DataSourceStatus/Initialized changes I commented above. 

If the SDK is in "offline" mode, that means the data system + events are disabled. The SDK isn't making any external connections, to a database or LaunchDarkly.